frontend/src/graphql/queries.js

```diff
@@ -28,6 +28,15 @@ export const authenticateGmsUserSearch = gql`
   }
 `
 
+export const userLocationQuery = gql`
+  query {
+    userLocation {
+      userLocation
+      communityLocation
+    }
+  }
+`
+
```

Comment:
This is the query, which is not servered by the backend till now, but it is only used in the User-settings dialog with the gms user-location capturing.